lib/std/jsfetch.nim

```diff
@@ -14,12 +14,12 @@ when defined(nimExperimentalJsfetch) or defined(nimdoc):
       metod* {.importjs: "method".}: cstring
       body*, integrity*, referrer*, mode*, credentials*, cache*, redirect*, referrerPolicy*: cstring
 
-    FetchModes* = enum  ## JavaScript Fetch API mode options.
+    FetchModes* = enum  ## Mode options.
       fmCors = "cors"
       fmNoCors = "no-cors"
       fmSameOrigin = "same-origin"
 
-    FetchCredentials* = enum  ## JavaScript Fetch API Credential options.
+    FetchCredentials* = enum  ## Credential options.
```

Comment:
```suggestion
    FetchCredentials* = enum  ## Credential options. See https://developer.mozilla.org/en-US/docs/Web/API/Request/credentials
```

likewise, can you add links for other types?

Reply:
I tried to add them but not all has their own links.